communication/calendar-guidelines.md

```diff
@@ -95,6 +100,54 @@ will prevent you from getting updates like a canceled meeting. Join the main
 
 Accept the invite from the sender and you'll have the updates.
 
+### Calendar event template
+
+In case you are creating a calendar event and wondering what to include in it, you can refer to the following template
+
+#### Title
+
+```
+[SIG-<name>] <meeting title>
+or
+[WG-<name>] <meeting title>
+or
+[User Group-<name>]
+```
+#### Description
+
+```
+<One line intro about the SIG/ WG. Can include github links>
+
+Time: <Weekly/ biweekly/... or is the meeting for a fixed time and till when?>
+
+Meeting Notes & Agenda: <Hyperlink to the meeting notes doc>
+
+Zoom link: <Zoom invite link. Preferably both app and web client links>
+
+Additional Info: <Any extra info/ links can be provided here>
+
+Livestream link: <optional>
+
+Youtube Playlist: <optional>
+
+Google group: <optional>
+```
+
+#### Example
+
+> SIG contribex sub project, Contributor Marketing team weekly discussion
+> 
+> https://github.com/kubernetes/community/tree/master/communication/marketing-team
+> 
+> **Time**: Weekly. Every Friday
+> 
+> **Meeting Notes & Agenda**: https://docs.google.com/document/d/1IlHAJ131akGhI5ffF4OoVW0PrsVY4C0BB8l-UyQaQVo/edit
+> 
+> **Zoom link**:
+>
+> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; App: https://zoom.us/j/596959769
+>
+> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Web Client: https://zoom.us/wc/join/596959769
```

Comment:
```suggestion
```

Seems redundant

Reply:
The web client URL is for folks who can't use the full blown client, it makes it easier for them to join without having to find it themselves 👍 The one thing I'd do (outside of trimming the `&nbsp`) is replace the number with `XXXXXXXXXX`

Reply:
Yeah, I agree with adding a separate web client and app client URL